build-decision/src/build_decision/repository.py

```diff
@@ -45,9 +47,9 @@ def get_file(self, path, *, revision=None):
             if revision is None:
                 revision = "master"  # FIXME: Use api to get default branch
             if repo_url.startswith("https://github.com/"):
-                if repo_url.endswith("/"):
-                    repo_url = repo_url[:-1]
-                url = f"{repo_url}/raw/{revision}/{path}"
+                url = f"https://raw.githubusercontent.com/{self.repo_path}/{revision}/{path}"
```

Comment:
Should we use the documented API (https://docs.github.com/en/rest/repos/contents?apiVersion=2022-11-28) here instead of the githubusercontent domain?

Reply:
Yeah probably, I didn't think about checking if they had an API. I'll test it and change it if it works

Reply:
As far as I can tell you can get the raw contents with `Accept: application/vnd.github.raw+json`?

Reply:
You're right... I only looked at their examples which doesn't list the raw one... I'll change it